### PR TITLE
om= now interprets theme (eco) settings

### PR DESCRIPTION
**Pull request by mkopec3**
Done according to Maijin's suggestion from PR #13036 

**Before the change:**
omeq_before

**After the change:**
omeq_after

### CODE REVIEW

**Comment by Maijin** [comment]
Great! see also `o=` this one doesn't have color at all.

**Comment by radare** [comment]
maybe it will be nice to also colorize the '#'

File: libr/core/cmd_open.c [hunk 1]
```diff
@@ -205,10 +205,12 @@ static void list_maps_visual(RIO *io, ut64 seek, ut64 len, int width, int use_co
...
 				color_end = Color_RESET;
-				if (s->perm & R_PERM_X) { // exec bit
-					color = Color_GREEN;
+				if ((s->perm & R_PERM_X) && (s->perm & R_PERM_W)) { // exec & write bits
+					color = r_cons_singleton()->context->pal.comment;
+				} else if (s->perm & R_PERM_X) { // exec bit
+					color = r_cons_singleton()->context->pal.args;
 				} else if (s->perm & R_PERM_W) { // write bit
-					color = Color_RED;
+					color = r_cons_singleton()->context->pal.input;
```
Comment (radare) [inline libr/core/cmd_open.c]: i think the colors should be:

is_writable = graph.false # red by default
is_executable = graph.true # green by default
is_exec+is_writable = graph.trufae # blue by default

**Comment by mkopec3** [comment]
Done.
Adjusted this also for iS= (for consistency purpose)
However I would propose to leave _widget_sel_ for w/x on sections, just to highlight the fact that it's really bad when a section has those both permissions set together (e.g. executable stack)

**Comment by mkopec3** [comment]
Regarding '#' colorization, I can adjust it later

**Comment by radare** [comment]
ok

**Pull request merged**